kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -73,13 +73,28 @@ data:
         # An array of Kubernetes services to scrape metrics from.
         # kubernetes_services = ["http://my-service-dns.my-namespace:9102/metrics"]
 
-        # When monitor_kubernetes_pods = true, replicaset will scrape Kubernetes pods for the following prometheus annotations:
-        # - prometheus.io/scrape: Enable scraping for this pod
-        # - prometheus.io/scheme: If the metrics endpoint is secured then you will need to
-        #     set this to `https` & most likely set the tls config.
-        # - prometheus.io/path: If the metrics path is not /metrics, define it with this annotation.
-        # - prometheus.io/port: If port is not 9102 use this annotation
-        monitor_kubernetes_pods = false
+        ## Enable replicaset scraping of k8s pods. Further settings as to which pods to scape are determined by the 'method' option below. When enabled, the default is to use annotations to determine whether to scrape or not.
+        # monitor_kubernetes_pods = false
+        
```

Comment:
I think, we can just upgrade telegraf  as part of this update .Lets not make config updates considering the metrics will be supported through metrics addon and we dont have customers are asking for these features.  For adding the config changes, will have to test all the config changes and scale etc.

Reply:
updated code